lib/spaceship/tunes/tunes_client.rb

```diff
@@ -393,6 +393,36 @@ def price_tier(app_id)
       end
     end
 
+    # Returns an array of all available pricing tiers
+    # @return [Array] the PricingTier objects
+    # [{
+    #   "tierStem": "0",
+    #   "tierName": "Free",
+    #   "pricingInfo": [{
+    #       "country": "United States",
+    #       "countryCode": "US",
+    #       "currencySymbol": "$",
+    #       "currencyCode": "USD",
+    #       "wholesalePrice": 0.0,
+    #       "retailPrice": 0.0,
+    #       "fRetailPrice": "$0.00",
+    #       "fWholesalePrice": "$0.00"
+    #     }, {
+    #     ...
+    # }, {
+    # ...
+    def pricing_tiers
```

Comment:
I meant by the library. Is it possible to do 

```
client = Spaceship::TunesClient.new
client.pricing_tiers
````
Answer: it's not:
```
RuntimeError: Please login using `Spaceship::Tunes.login('user', 'password')`
	from /Users/lacostej/Code/OSS/Fastlane2/countdown/spaceship/lib/spaceship/tunes/tunes_base.rb:9:in `client'
	from /Users/lacostej/Code/OSS/Fastlane2/countdown/spaceship/lib/spaceship/base.rb:181:in `initialize'
	from /Users/lacostej/Code/OSS/Fastlane2/countdown/spaceship/lib/spaceship/tunes/pricing_tier.rb:22:in `new'
```

I suspect this is more a problem in the design of spaceship, which assumes being logged in is the first thing to do, and which has a strange way of accessing the current client. So you could let this off for now, maybe just document it in the method (something like "Although this information is publicly available, spaceship requires you to have a logged in client for now to access it"). It's somewhat related to #219